tests/bootstrap.php

```diff
@@ -19,20 +20,20 @@
 // define for use in WP bootstrap file
 define('WP_TESTS_CONFIG_FILE_PATH', $currentTestEnvironment->config());
 
-// pull in wp test functions like tests_add_filter
-require_once $currentTestEnvironment->functions();
-
 // load GiveWP
-tests_add_filter('muplugins_loaded', static function () {
+TestHooks::addFilter('muplugins_loaded', static function () {
```

Comment:
What's the difference in using the new `TestHooks` over `tests_add_filter`?

Reply:
They are the same thing functionally.  We were pulling in _all_ the WP test suite functions in this file just to use that `tests_add_filter` function.  It was also tricky because that functions file is a dynamic path depending on what environment you are in (local vs workflow).  Also, the main WP bootstrap file being included at the end _already_ includes that functions file.  So `TestHooks` pulls out the things we need like addFilter and can be used in add-ons without double loading WP's functions file.